### PR TITLE
remove submit button if already submitted

### LINKED ISSUE #1316: Resubmit same report by going back
Hey Everyone,
Maybe this has been raised already but on one of my experiments using the product I noticed that after I received my confirmation number on the confirmation page I can hit back in the browser and either re-submit or forward again and it will re-submit the report again.

Maybe after we submit we want to clear the data?

### LINKED ISSUE #1600: Cannot submit a report after cancel page if the victim use "<-" to go to previous page
## Summary

Cannot submit a report after cancel page if the victim use "<-" to go to previous pages instead of click on "Create a new report"

## Steps to reproduce
1. Launch report and fill out the form
2.  At summary page, the user wants to cancel the report
3. After cancel report, the user wants to fill out the report again but he/she uses "<-" key to get back to summary page and edit each session to enter information instead of click on "Create a new report" option 
4. The user receives the message "Submitting your report ..." at the Thank you page and after 2 minutes, the user receive an message "Sorry, your report was not successfully submitted"

Here is my concern
1. Currently, we allow the user to submit their report again if they use "<-" key but it does not work if the user cancels their report and want to submit it again with "<-" key
2. Why does it take so long (2 min) to receive an error message

thankYou

## Unresolved questions

> Are there any related issues you consider out of scope for this issue that could be addressed in the future?

### PR DESCRIPTION
**Pull request by sastels**
Fixes #1600 
Fixes #1316 

# Description

If the user submits, goes to the thankyou page, and clicks the back button, put a thank you message on the confirmation page instead of the submit button.

image

# Checklist:

- [x] I have looked at my code on GitHub and it all looks good (ex: no random commented out code or console.logs)
- [x] I have added and needed tests for my changes (in particular for new screens)
- [x] I have added a comment to any confusing code
- [x] I have added documentation to any modified front-end code. (Or added missing documentation)

### CODE REVIEW

**Comment by sastels** [comment]
Now
image

File: f2/server.js [hunk 1]
```diff
@@ -167,10 +167,10 @@ app
     res.sendFile(path.join(__dirname, 'build', 'index.html'))
   })
 
-// uncomment to allow direct loading of arbitrary pages
-// .get('/*', function(_req, res) {
-//   res.sendFile(path.join(__dirname, 'build', 'index.html'))
-// })
+  // uncomment to allow direct loading of arbitrary pages
```
Comment (amazingphilippe) [inline f2/server.js]: Is this intentional for the staging server?

Comment (sastels) [reply f2/server.js]: whoops! good catch!

Comment (sastels) [reply f2/server.js]: good catch!

**Pull request merged**